sematic/api/endpoints/resolutions.py

```diff
@@ -261,3 +263,21 @@ def cancel_resolution_endpoint(
     )
 
     return flask.jsonify(dict(content=resolution.to_json_encodable()))
+
+
+@sematic_api.route(
+    "/api/v1/resolutions/<resolution_id>/external_resources", methods=["GET"]
+)
+@authenticate
+def get_resources_endpoint(user: Optional[User], resolution_id: str) -> flask.Response:
+    resource_ids = get_resource_ids_by_root_id(resolution_id)
+    resources = []
+    for resource_id in resource_ids:
+        resource = get_external_resource_record(resource_id)
+        assert resource is not None  # should be impossible since ids came from the DB
+
+        # TODO: update state if refresh=True is provided as a parameter and
+        # the state for the resource is managed remotely.
```

Comment:
My suggestion was to do that on `GET /api/v1/external_resources/<id>`, not on the bulk endpoint.

Reply:
yes, forgot to remove this comment after our convo. Removed now